google-cloud-bigquery-storage/src/main/resources/reference.conf

```diff
@@ -0,0 +1,13 @@
+alpakka.google.cloud.bigquery.grpc {
+  host = "bigquerystorage.googleapis.com"
+  port = 443
+
+  # Set to "none" to disable TLS
+  # TLS and certificates should be configured via the underlying
+  # SSL-config library
+  # https://lightbend.github.io/ssl-config/WSQuickStart.html#point-the-trust-manager-at-the-pem-file
+  # rootCa = "none"
+
+  credentials.default-scopes = ${?alpakka.google.credentials.default-scopes} ["https://www.googleapis.com/auth/bigquery.readonly"]
```

Comment:
This is at the wrong path.
Currently `alpakka.google.cloud.bigquery.grpc.credentials.default-scopes`
Should be `alpakka.google.credentials.default-scopes` (to match the override)